src/chocolatey.tests.integration/chocolatey.tests.integration.csproj

```diff
@@ -219,6 +219,9 @@
     <None Include="context\installpackage\1.0.0\tools\Casemismatch.exe.ignore">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="context\installpackage\1.0.0\tools\chocolateyBeforeModify.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>

```

Comment:
I believe so. Scripts are only run if they are present. I imagine we should verify that no attempt is made to run the modification script on initial installation.